Domain/Models/Floor.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Models
+{
+    public class Floor : BaseEntity
+    {
+        public int NumPlaces { get; set; }
+        public int FloorNum { get; set; }
```

Comment:
I suggest to avoid have some cut name, FloorNumber or just Floor will be more understandable. same for NumPlace, 
1 do we really need this field, if we are able to count them by Count(x=> x.Seats) ?
2 if you want to denormalize Database, it's acceptable, in such case I suggest SeatsAmount for name, but for our internship I suggest to keep database normalization forms almost everywhere.